tests/publiccloud/nvidia.pm

```diff
@@ -18,24 +18,46 @@ sub validate_nvidia {
     validate_script_output("hwinfo --gfxcard", sub { /nVidia.*Tesla T4/mg });
     # Check loaded modules
     assert_script_run("lsmod | grep nvidia", fail_message => "nvidia module not loaded");
-    # Ideally, we should check for nvidia-smi here but it is delivered by nvidia-compute-utils-G06 package, which is not available on PublicCloud
+    assert_script_run("nvidia-smi");
```

Comment:
Could you please capture output of `nvidia-smi`? Would be nice to use this output in recordinfo and also do some perl regex on it.